Reactive Planners Build doesn't work

Hello,

I try to install packages on my computer, but I couldn't build dependencies. Here is the result of `colcon build --cmake-args -DCMAKE_BUILD_TYPE=Release`

You can find the events.log and logger_all.log for details.

Ubuntu 18.04
Anaconda Python 3.6



```
Starting >>> googletest-distribution
Starting >>> mpi_cmake_modules
Starting >>> pybind11
--- stderr: mpi_cmake_modules                                                
CMake Error at cmake/sphinx.cmake:65 (message):
  breathing-cat not found! Please install using: pip3 install breathing-cat
Call Stack (most recent call first):
  cmake/documentation.cmake:43 (add_sphinx_documentation)
  CMakeLists.txt:85 (add_documentation)


---
Failed   <<< mpi_cmake_modules [0.66s, exited with code 1]
Aborted  <<< pybind11 [0.81s]                    
Aborted  <<< googletest-distribution [1.01s]                            

Summary: 0 packages finished [1.62s]
  1 package failed: mpi_cmake_modules
  2 packages aborted: googletest-distribution pybind11
  2 packages had stderr output: mpi_cmake_modules pybind11
  7 packages not processed

[events.log](https://github.com/machines-in-motion/reactive_planners/files/9952151/events.log)
[logger_all.log](https://github.com/machines-in-motion/reactive_planners/files/9952152/logger_all.log)
```


When I try to install breathing-cat: `pip3 install breathing-cat`

```
ERROR: Could not find a version that satisfies the requirement breathing-cat (from versions: none)
ERROR: No matching distribution found for breathing-cat
```

--- a/cmake/sphinx.cmake
+++ b/cmake/sphinx.cmake
@@ -46,45 +46,48 @@ macro(ADD_SPHINX_DOCUMENTATION)
     # year or so.
   endif()
 
-  # TODO: This should probably be handled in a nicer way that is more obvious to
-  # the user (probably via an argument that has to be passed?
-  if (DEFINED PYTHON_INSTALL_DIR)
-    set(PYTHON_PACKAGE_LOCATION ${CMAKE_INSTALL_PREFIX}/${PYTHON_INSTALL_DIR}/${PROJECT_NAME})
-  else()
-    set(PYTHON_PACKAGE_LOCATION ${PROJECT_SOURCE_DIR}/python/${PROJECT_NAME})
-  endif()
-
-  # Build and install directories
-  set(SPHINX_DOC_BUILD_FOLDER ${CMAKE_BINARY_DIR}/share/docs/sphinx)
-  set(SPHINX_DOC_INSTALL_FOLDER share/${PROJECT_NAME}/docs/sphinx)
-
-
-  # make sure bcat is installed
-  find_program(BCAT bcat)
-  if(NOT BCAT)
-    message(FATAL_ERROR
-        "breathing-cat not found! "
-        "Please install using: pip3 install breathing-cat"
-    )
-  endif()
-
-  # Create the output
-  add_custom_target(
-    ${PROJECT_NAME}_sphinx_html
-    ${BCAT} --package-dir "${PROJECT_SOURCE_DIR}"
-        --output-dir "${SPHINX_DOC_BUILD_FOLDER}"
-        --python-dir "${PYTHON_PACKAGE_LOCATION}"
-        --force
-    COMMENT "Building documentation for ${PROJECT_NAME}"
-  )
-
-  # install the documentation
   if(GENERATE_DOCUMENTATION)
+
+    # TODO: This should probably be handled in a nicer way that is more obvious
+    # to the user (probably via an argument that has to be passed?
+    if (DEFINED PYTHON_INSTALL_DIR)
+      set(PYTHON_PACKAGE_LOCATION
+          ${CMAKE_INSTALL_PREFIX}/${PYTHON_INSTALL_DIR}/${PROJECT_NAME})
+    else()
+      set(PYTHON_PACKAGE_LOCATION ${PROJECT_SOURCE_DIR}/python/${PROJECT_NAME})
+    endif()
+
+    # Build and install directories
+    set(SPHINX_DOC_BUILD_FOLDER ${CMAKE_BINARY_DIR}/share/docs/sphinx)
+    set(SPHINX_DOC_INSTALL_FOLDER share/${PROJECT_NAME}/docs/sphinx)
+
+
+    # make sure bcat is installed
+    find_program(BCAT bcat)
+    if(NOT BCAT)
+      message(FATAL_ERROR
+          "breathing-cat not found! "
+          "Please install using: pip3 install breathing-cat"
+      )
+    endif()
+
+    # Create the output
+    add_custom_target(
+      ${PROJECT_NAME}_sphinx_html
+      ${BCAT} --package-dir "${PROJECT_SOURCE_DIR}"
+          --output-dir "${SPHINX_DOC_BUILD_FOLDER}"
+          --python-dir "${PYTHON_PACKAGE_LOCATION}"
+          --force
+      COMMENT "Building documentation for ${PROJECT_NAME}"
+    )
+
+    # install the documentation
     install(DIRECTORY ${SPHINX_DOC_BUILD_FOLDER}/html
             DESTINATION ${SPHINX_DOC_INSTALL_FOLDER})
-  endif()
 
-  # Create a dependency on the doc target
-  add_dependencies(doc ${PROJECT_NAME}_sphinx_html)
+    # Create a dependency on the doc target
+    add_dependencies(doc ${PROJECT_NAME}_sphinx_html)
+
+  endif()
 
 endmacro(ADD_SPHINX_DOCUMENTATION)

